### PR TITLE
Add Joy Mod axolotl variants using MoreAxolotlVariantsAPI

### PR DESCRIPTION
**Pull request by rotgruengelb**

### CODE REVIEW

**Comment by NormalFery** [comment]
LGTM, but I'd wait for blurry to say regardless

File: src/main/resources/assets/minecraft/texts/credits.json [hunk 1]
```diff
@@ -8837,7 +8837,8 @@
                         "KTrain",
                         "Santiago 'Zhamty' Navas",
                         "TheClashFruit",
-                        "Wolren"
+                        "Wolren",
+                        "rotgruengelb"
```
Comment (blryface) [inline src/main/resources/assets/minecraft/texts/credits.json:8841]: Alphabetical Order

**Review by blryface — CHANGES_REQUESTED** [review]
lgtm apart from the credits.json order thignngngn

**Review by blryface — APPROVED** [review]
what the hell is a Pride Craft
(I'm dislexic blurry)

**Pull request merged**